R/tmb.R

```diff
@@ -567,8 +567,8 @@ h_mmrm_tmb_fit <- function(tmb_object,
 #' @examples
 #' formula <- FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID)
 #' data <- fev_data
-#' system.time(result <- h_mmrm_tmb(formula, data, rep(1, nrow(fev_data))))
-h_mmrm_tmb <- function(formula,
+#' system.time(result <- fit_mmrm(formula, data, rep(1, nrow(fev_data))))
```

Comment:
since now this is called `fit_*` and does not need TMB understanding, would it make sense to move to `fit.R`?... not necessary just wondering

Reply:
moved.